cms/static/js/spec/views/programs/program_creator_spec.js

```diff
@@ -122,6 +123,25 @@ define([
                 expect( view.saveError ).not.toHaveBeenCalled();
             });
 
+            it( 'should submit the form correctly when creating micromasters program ', function(){
+                var programId = 221,
+                    micromasterProgramInput = $.extend({}, sampleInput);
```

Comment:
Suggestion: move the definition of `sampleInput` to beforeEach so that you don't need to perform this extension when you want to change it.